files/mcollective/agent/r10k.json

```diff
@@ -0,0 +1,308 @@
+        }
+      },
+      "output": {
+        "path": {
+          "description": "Operating on push",
+          "display_as": "Path",
+          "default": null
+        },
+        "output": {
+          "description": "Output from git",
+          "display_as": "Output",
+          "default": null
+        },
+        "error": {
+          "description": "Error from git",
+          "display_as": "Errors",
+          "default": null
+        }
+      },
+      "display": "always",
+      "description": "Push "
+    },
+    {
+      "action": "status",
+      "input": {
+        "path": {
+          "prompt": "Module path",
+          "description": "Operating on status",
+          "type": "string",
+          "default": null,
+          "optional": false,
+          "validation": ".",
+          "maxlength": 256
+        },
+        "user": {
+          "prompt": "User",
+          "description": "User to run as",
+          "type": "string",
+          "default": null,
+          "optional": true,
+          "validation": "\\w+",
+          "maxlength": 32
+        }
+      },
+      "output": {
+        "path": {
+          "description": "Operating on status",
+          "display_as": "Path",
+          "default": null
+        },
+        "output": {
+          "description": "Output from git",
+          "display_as": "Output",
+          "default": null
+        },
+        "error": {
+          "description": "Error from git",
+          "display_as": "Errors",
+          "default": null
+        }
+      },
+      "display": "always",
+      "description": "Status "
+    },
+    {
+      "action": "sync",
+      "input": {
+        "user": {
+          "prompt": "User",
+          "description": "User to run as",
+          "type": "string",
+          "default": null,
+          "optional": true,
+          "validation": "\\w+",
+          "maxlength": 32
+        }
+      },
+      "output": {
+        "output": {
+          "description": "Output from git",
+          "display_as": "Output",
+          "default": null
+        },
+        "error": {
+          "description": "Error from git",
+          "display_as": "Errors",
+          "default": null
+        }
+      },
+      "display": "always",
+      "description": "Sync "
+    },
+    {
+      "action": "synchronize",
+      "input": {
+        "user": {
+          "prompt": "User",
+          "description": "User to run as",
+          "type": "string",
+          "default": null,
+          "optional": true,
+          "validation": "\\w+",
+          "maxlength": 32
+        }
+      },
+      "output": {
+        "output": {
+          "description": "Output from git",
+          "display_as": "Output",
+          "default": null
+        },
+        "error": {
+          "description": "Error from git",
+          "display_as": "Errors",
+          "default": null
+        }
+      },
+      "display": "always",
+      "description": "Synchronize "
+    }
+  ]
+}
```

Comment:
can you please add the missing newline at the end?

Reply:
Fixed